evaluation.py

```diff
@@ -0,0 +1,87 @@
+import os
+import argparse
+import pandas as pd
+from omegaconf import OmegaConf
+
+from text_models import W2VModel, FastTextModel, BertModelMLM, SBertModel, RandomEmbeddingModel
+from data_processing.util import get_corpus, CONFIG_PATH
+from approaches import SimpleApproach, TfIdfApproach, IntersectionApproach
+
+
+def parse_arguments():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--topn",
+        dest="topn",
+        action="store",
+        type=int,
+        default=5,
+        help="The number of predicted duplicate bug-reports for one report",
+    )
+    parser.add_argument("--w2v", dest="w2v", action="store_true", help="Use word2vec model for classification")
+    parser.add_argument(
+        "--fasttext", dest="fasttext", action="store_true", help="Use fasttext model for classification"
+    )
+    parser.add_argument("--bert", dest="bert", action="store_true", help="Use BERT model for classification")
+    parser.add_argument("--sbert", dest="sbert", action="store_true", help="Use SBERT model for classification")
+    parser.add_argument("--random", dest="random", action="store_true", help="Use random embeddings for classification")
+    parser.add_argument(
+        "--intersection",
+        dest="intersection",
+        action="store_true",
+        help="Use word intersections for success rate evaluation",
+    )
+    parser.add_argument("--tfidf", dest="tfidf", action="store_true", help="Use tf-idf matrix in evaluation")
+    return parser.parse_args()
```

Comment:
Evaluation setup should be loaded from the config file

Reply:
Moved to `config.yml`